components/admin_console/license_settings/starter_edition/starter_left_panel.tsx

```diff
@@ -73,14 +75,21 @@ const StarterLeftPanel: React.FC<StarterEditionProps> = ({
                 <div className='uploadButtons'>
                     <button
                         className='btn btn-upload light-blue-btn'
-                        onClick={openUploadModal}
+                        onClick={() => fileInputRef.current !== null && fileInputRef.current.click()}
                         id='open-modal'
                     >
                         <FormattedMessage
                             id='admin.license.uploadFile'
                             defaultMessage='Upload File'
                         />
                     </button>
+                    <input
+                        ref={fileInputRef}
+                        type='file'
+                        accept='.mattermost-license'
```

Comment:
This is also used in `upload_license_modal.tsx`, should be a constant config?

Reply:
it is a local constant in upload_license_modal, you are right, it is better if we set it as a global constant.